docs/install.rst

```diff
@@ -393,6 +393,22 @@ packages that may be useful for radio astronomy).
    <https://github.com/astropy/astropy/issues?q=+label%3ACASA-Installation+>`_
    and if you do not encounter your issue there, please post a new one.
 
+
+Installing pre-built Development Versions of ``astropy``
+--------------------------------------------------------
+
+Most nights a development snapshot of ``astropy`` will be compiled.
```

Comment:
Do you think it is useful to provide a link for downstream developers to check the build logs? For example, personally, I would like to see when did the last "nightly" happen.

Also, is this available for all OSes? Or just Linux?

Reply:
All the binaries that are built for a release are built. You can see the list of available versions (and their upload date) here: https://dev.azure.com/astropy-project/astropy/_packaging?_a=package&feed=nightly&package=astropy&protocolType=PyPI&version=4.3.dev694&view=versions

Reply:
Should we add that link to the doc?